nltk/metrics/distance.py

```diff
@@ -22,6 +22,8 @@
 import operator
 import warnings
 
+import stringzilla as sz
```

Comment:
Depending on how widespread we intend to rely on stringzilla, it may make sense to move the import to inside of the `edit_distance` method to avoid unnecessary importing.

Reply:
Hi @tomaarsen! Can you please double check the updated variant?